File: Manifest-Son/count_zeros.js
// Writing a program that takes an integer as input and return an integer with reversed digit ordering.
function reverseInterger(num){
    let reversed = 0;
    let remainder = Math.abs(num);
    // Math.abs() is used to convert negative numbers to positive

    while(remainder > 0){
        reversed = (reversed * 10) + (remainder % 10);
        remainder = Math.floor(remainder / 10);
    } 

    // While loop is used to reverse the number
    return num < 0 ? -reversed : reversed;
}

console.log(reverseInterger(123));

File: kennedymwaniki/count_zeros.py
def power_of_two(num):
    if num < 0:
        return False
    return (num & (num - 1)) == 0

print(power_of_two(8))  
print(power_of_two(7)) 
print(power_of_two(16))  
print(power_of_two(1)) 